x-pack/legacy/plugins/file_upload/public/index.ts

```diff
@@ -4,4 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { JsonUploadAndParse } from './components/json_upload_and_parse';
+import { FileUploadPlugin as Plugin } from './plugin';
+
+export function plugin() {
+  return new Plugin();
```

Comment:
no need to alias, just do `new FileUploadPlugin`

Even beyond that, what is this file adding beyond indirection?

Reply:
Yeah, that's about all it's adding here since we're not sharing any [static code](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#2-export-all-static-code-and-types-from-publicindexts). I threw it in there to give it the same "shape" as other plugins. Other examples I saw were pretty minimal and didn't add much either. Agreed that aliasing is superfluous here.